src/ring_theory/noetherian.lean

```diff
@@ -508,3 +517,93 @@ nat.rec_on n
 (λ n ih, by simpa [pow_succ] using fg_mul _ _ h ih)
 
 end submodule
+
+namespace primes
```

Comment:
```suggestion
section primes
```
I don't expect name clashes, so putting this in a namespace just makes the names longer, without really helping.